### PR TITLE
Add com.obsproject.Studio.Plugin.CompositeBlur

### PR DESCRIPTION
**Pull request by dennisrijsdijk**
### Please confirm your submission meets all the criteria

- [X] Please describe the application briefly.
  
  A comprehensive blur plugin for OBS that provides several different blur algorithms, and proper compositing.

- [X] The domain used for the application ID is [controlled by the application developers][appid-domain] and the [application id guidelines][appid] are followed.
- [X] I have read the and followed all the [Submission and licence requirements][reqs].
- [X] I have [built][build] and tested the submission locally.
- [X] I am an author/developer/upstream contributor of the project. If not, I contacted upstream developers about this submission. **Link:** https://github.com/FiniteSingularity/obs-composite-blur/issues/102#issuecomment-2323009426

[appid-domain]: https://docs.flathub.org/docs/for-app-authors/requirements/#control-over-domain-or-repository
[appid]: https://docs.flathub.org/docs/for-app-authors/requirements#application-id
[reqs]: https://docs.flathub.org/docs/for-app-authors/requirements
[build]: https://docs.flathub.org/docs/for-app-authors/submission/#before-submission

### CODE REVIEW

File: com.obsproject.Studio.Plugin.CompositeBlur.yaml [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+modules:
+  - name: composite-blur
+    buildsystem: cmake-ninja
+    config-opts:
+      - -DCMAKE_BUILD_TYPE=Release
+      - -DLINUX_PORTABLE=off
+    post-install:
+      - install -Dm644 --target-directory=${FLATPAK_DEST}/share/metainfo ${FLATPAK_ID}.metainfo.xml
+      - appstream-compose --basename=${FLATPAK_ID} --prefix=${FLATPAK_DEST} --origin=flatpak
+        ${FLATPAK_ID}
```
Comment (bbhtt) [inline com.obsproject.Studio.Plugin.CompositeBlur.yaml:20]:
```suggestion
```

File: com.obsproject.Studio.Plugin.CompositeBlur.yaml [hunk 2]
```diff
@@ -0,0 +1,30 @@
+id: com.obsproject.Studio.Plugin.CompositeBlur
+branch: stable
+runtime: com.obsproject.Studio
+runtime-version: stable
+sdk: org.kde.Sdk//6.6
+build-extension: true
+separate-locales: false
+appstream-compose: false
```
Comment (bbhtt) [inline com.obsproject.Studio.Plugin.CompositeBlur.yaml:8]:
```suggestion
```

File: flathub.json [hunk 3]
```diff
@@ -0,0 +1,4 @@
+{
+  "skip-icons-check": true,
+  "onl-arches": ["x86_64"]
```
Comment (bbhtt) [inline flathub.json:3]:
```suggestion
  "only-arches": ["x86_64"]
```

Comment (bbhtt) [reply flathub.json:3]: why this restriction?

Comment (hfiguiere) [reply flathub.json:3]: OBS is currently x86_64 only

**Comment by bbhtt** [comment]
bot, build com.obsproject.Studio.Plugin.CompositeBlur

**Comment by dennisrijsdijk** [comment]
App from testing repo installs and runs as expected

**Pull request closed without merging**